docs/alluxio-with-minio.md

```diff
@@ -1,19 +1,19 @@
-# Deploy Alluxio with Minio [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+# Deploy Alluxio with MinIO [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
```

Comment:
As @kannappanr mentioned, the links here and below can be changed to `min.io` instead of `minio.io`

Reply:
I was doing it in stages. I will send another PR for it immediately.